lib/broccoli/amd-shim.js

```diff
@@ -6,11 +6,14 @@ module.exports = function shimAmd(tree, nameMapping) {
     let name = nameMapping[relativePath];
     if (name) {
       return [
-        '(function(define){\n',
+        '(function(define, module){\n',
```

Comment:
Right, but in that case it is only the first arg (not an arbitrary number of leading arguments), so I don't think we need a loop there...